hasura.planx.uk/migrations/1665744811194_delete_lowcal_events_on_submit_or_soft_delete/up.sql

```diff
@@ -0,0 +1,23 @@
+DROP TRIGGER IF EXISTS delete_lowcal_session_scheduled_events_trigger ON lowcal_sessions;
+DROP FUNCTION IF EXISTS delete_lowcal_session_scheduled_events;
+
+CREATE OR REPLACE FUNCTION 
+	delete_lowcal_session_scheduled_events()
+RETURNS TRIGGER AS $$ 
+BEGIN
+	DELETE FROM hdb_catalog.hdb_scheduled_events 
+	WHERE comment IN (
+    'reminder_' || OLD.id, 
+		'expiry_' || OLD.id 
+	);
+	RETURN NULL;
+END
+$$ LANGUAGE plpgsql VOLATILE;
+
+DROP TRIGGER IF EXISTS delete_lowcal_session_scheduled_events_trigger ON lowcal_sessions;
+CREATE TRIGGER delete_lowcal_session_scheduled_events_trigger
+AFTER UPDATE OF submitted_at, deleted_at ON lowcal_sessions FOR EACH ROW
```

Comment:
This line has the actual change - we should delete these events on update of `submitted_at` or `deleted_at`, not just when the row is deleted.